src/ffi.rs

```diff
@@ -408,36 +411,43 @@ mod fuzz_dummy {
     // Contexts
     /// Creates a dummy context, tracking flags to ensure proper calling semantics
     pub unsafe fn secp256k1_context_preallocated_create(_ptr: *mut c_void, flags: c_uint) -> *mut Context {
+        assert!(UNSAFE_CRYPTO_FUZZING, "Tried fuzzing without setting the UNSAFE_CRYPTO_FUZZING variable");
```

Comment:
A constant for the string seems to be a good idea. Maybe even a function / macro `assert_fuzzing()`. (Well it's a one-liner, so it does not make much of a difference if it's a function call or the assert directly. I think both ways are fine.)